AppRentCar/tests.py

```diff
@@ -1,3 +1,54 @@
+from django.contrib.auth.models import User
 from django.test import TestCase
+from django.utils import timezone
+from .models import Rent, RentalTerms, Car, CompanyBranches
 
-# Create your tests here.
+
+class RentAvailabilityTest(TestCase):
+
+    def setUp(self):
+        self.user = User.objects.create_user(username='testuser', password='testpassword')
+        self.car = Car.objects.create(
+            brand='Brand',
+            model='Model',
+            cars_type='Suv',
+            engine='Benzyna',
+            capacity=2.0,
+            year='2023',
+            number_of_seats=5,
+            consumption='8L/100km',
+            power='150KM',
+            car_mileage='10 000 km',
+            transmission='Automatyczna',
+            no_gears='6',
+            drive='Przedni'
+        )
+        self.rental_terms = RentalTerms.objects.create(car=self.car, price=100.00)
+        self.start_date = timezone.now()
+        self.end_date = self.start_date + timezone.timedelta(days=5)
+
+    def test_car_available(self):
+        rent = Rent(rental_terms=self.rental_terms, start_date=self.start_date, end_date=self.end_date)
+        self.assertTrue(rent.is_car_available())
```

Comment:
tutaj dobrze byłoby sprawdzić kilka przypadków, jak jeszcze nikt nie zrobił rezerwacji, jak ktoś już zrobił na dane auto, ale my chcemy po nim wypożyczyć wtedy najlepiej wybrać zaraz po, żeby zobaczyć czy coś się nie zazębi, no i jak ktoś wypożycza inne auto, a my chcemy w tym samym terminie tylko inne, to takie pierwsze z brzegu przypadki które mi przyszły do głowy